app/src/main/java/org/oppia/android/app/options/AppLanguageActivityPresenter.kt

```diff
@@ -1,20 +1,34 @@
 package org.oppia.android.app.options
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
 import org.oppia.android.R
 import org.oppia.android.app.activity.ActivityScope
+import org.oppia.android.databinding.AppLanguageActivityBinding
 import javax.inject.Inject
 
 /** The presenter for [AppLanguageActivity]. */
 @ActivityScope
 class AppLanguageActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
   private lateinit var prefSummaryValue: String
 
-  fun handleOnCreate(prefKey: String, prefSummaryValue: String) {
-    activity.setContentView(R.layout.app_language_activity)
-    setLanguageSelected(prefSummaryValue)
+  fun handleOnCreate(prefKey: String, prefValue: String) {
+    val binding: AppLanguageActivityBinding = DataBindingUtil.setContentView(
+      activity,
+      R.layout.app_language_activity,
+    )
+    val toolbar = binding.appLanguageToolbar
+    toolbar.setNavigationOnClickListener {
```

Comment:
directly use the value binding.appLanguageToolbar instead of creating a variable

Reply:
Done